docs/source/science/NISAR/Simulated_NISAR.ipynb

```diff
@@ -0,0 +1,11823 @@
+{
```

Comment:
<p>Instead of, <code>If you would like ... click here.</code>, let's go with this (using "click here" is a hyperlink anti-pattern, as the linked text "here" is not descriptive of what it links to):</p><p> </p><p><code></code><code></p><p>For more information, see [Authentication: Before we can use earthaccess we need an account with NASA EDL](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/tutorials/earthaccess-demo.html#authentication-before-we-can-use-earthaccess-we-need-an-account-with-nasa-edl)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
That is noted - thanks! 🙂